### PR TITLE
Fixed some typos and improved formulations

### PR DESCRIPTION
**Pull request by vilemkrejcar**
As written in the name of the pull request, small text improvements were introduced.

### CODE REVIEW

File: slides/02/02.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ $$→w = (⇉X^T⇉X)^{-1}⇉X^T→t.$$
 
 Assume we want to predict a $t ∈ ℝ$ for a given $x ∈ ℝ$. If we train
 the linear regression with “raw” input vectors $→x = (x)$, only straight lines
-could be modeled.
+could be modelled.
```
Comment (akumm2k) [inline slides/02/02.md:32]: `modeled` is also correct.

Comment (foxik) [reply slides/02/02.md:32]: I go for US spelling when the US and British variants differ (I believe you see "modeling" and "modeled"  more often in papers). I will revert it once merged.

File: slides/02/02.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ Note that the $L^2$-regularization is usually not applied to the _bias_, only to
 ~~~
 Also, without penalizing the bias, linear regression with $L^2$-regularization is invariant
 to shifts (i.e., adding a constant to all the targets results in the same solution, only with
-the bias increased by that constant; if the bias was penalized, this would not
+the bias increased by that constant; if the bias were penalized, this would not
```
Comment (foxik) [inline slides/02/02.md:105]: You are the second one to suggest this, so I looked it up and learned how to use "was/were" in if clause of a conditional sentence -- thanks you all, I learned something new ;-)

**Comment by foxik** [comment]
Thanks, and another point.

**Pull request merged**